kernel/kernel-idgenerator-service/src/main/resources/logback.xml

```diff
@@ -1,12 +1,31 @@
 <configuration>
-	<springProperty scope="context" name="appName" source="spring.application.name"/>
-	<appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-		<encoder class="net.logstash.logback.encoder.LogstashEncoder"/>
+	<springProperty scope="context" name="appName"
+		source="spring.application.name" />
+	<appender name="STDOUT"
+		class="ch.qos.logback.core.ConsoleAppender">
+		<encoder class="net.logstash.logback.encoder.LogstashEncoder" />
 	</appender>
 
+	<appender name="ACCESS"
+		class="ch.qos.logback.core.ConsoleAppender">
+		<layout class="ch.qos.logback.classic.PatternLayout">
+			<Pattern>
+				%msg
+			</Pattern>
+		</layout>
+
+	</appender>
 	<root level="INFO">
 		<appender-ref ref="STDOUT" />
 	</root>
 
-	<logger name="io.vertx.core.impl.BlockedThreadChecker" level="OFF" />
+	<logger name="io.vertx.core.impl.BlockedThreadChecker"
+		level="OFF" />
+
+
+	<logger
+		name="io.mosip.kernel.idgenerator.config.AccessLogHandler" additivity="false">
+		<appender-ref ref="ACCESS" />
+	</logger>
+	l
```

Comment:
what is this l ? And why do we need these logback changes ?

Reply:
This is to remove the default pattern which is used for other logs as our json log was printed as a msg in that.

Reply:
As discussed please remove the I and test it once